.github/composite_actions/log_metric_wrapper/action.yaml

```diff
@@ -0,0 +1,103 @@
+name: Log CW Metric Wrapper
+description: Test to log a CW metric
+
+# Wraps log metric by configuring AWS credentials and adding github repo + run id inputs.
+inputs:
+  role-to-assume:
+    description: AWS role to assume for CW pushing
+    required: true
+  aws-region:
+    description: AWS region
+    required: true
+
+  # For getting failing step
+  job-status:
+    description: Used to determine if we track success or failure.
+    required: true
+  job-identifier:
+    description: For differentiating jobs of a run.
+    required: true
+  github-token:
+    required: true
+    description: Github token for requesting failing steps.
+
+  # Global Metric Dimensions
+  test-type:
+    description: canary, integration, unit testType.
+    required: true
+  category:
+    description: analytics, api, authenticator, etc.
+    required: true
+  workflow-name:
+    description: The Github Action workflow.yaml file name.  ie "AmplifyCanaries".
+    required: true
+
+  # FlutterDart Workflows Metric Dimensions
+  framework:
+    description: flutter, dart.
+    required: false
+  flutter-dart-channel:
+    description: beta, stable.
+    required: false
+  dart-version:
+    description: 3, 2.19, 2.18, etc.
+    required: false
+  flutter-version:
+    description: 3.10.6, 3.10.5, etc.
+    required: false
+  dart-compiler:
+    description: dart2js, ddc, dart, dart2wasm.
+    required: false
+
+  # Platform Workflows Metric Dimensions
+  platform:
+    description: android, ios, web, linux, windows.
+    required: false
+  platform-version:
+    description: ios-14.5, ios-16, android-25-x86, etc.
+    required: false
+runs:
+  using: "composite"
+  steps:
+    #- name: Exit if not scheduled
```

Comment:
We will uncomment and enforce tracking of scheduled success/failures when finalizing